src/CharacterState.js

```diff
@@ -6,38 +6,48 @@ export default class CharacterState {
     xPos: number; // Positive x is East
     yPos: number; // Positive y is South
     directionDegrees: number; // 0 is North, 90 is East
+    path: Array<any>;
```

Comment:
I think it would be worth adding a type for the path segments to `types.js`:

    export type PathSegment = {
        x1: number,
        y1: number,
        x2: number,
        y2: number
    };

And then defining `path` as `Array<PathSegment>`

Reply:
Actually, we probably don't need it to be in `types.js`. It's probably fine to make it local to the `CharacterState.js` file. Just add:

    type PathSegment = {
        x1: number,
        y1: number,
        x2: number,
        y2: number
    };

to `CharacterState.js`. The way we do for React component property types. I don't think we need the `PathSegment` type independent of `CharacterState`.